packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -849,19 +857,17 @@ - (void)stopVideoRecordingWithCompletion:(void (^)(NSString *_Nullable,
   if (_isRecording) {
     _isRecording = NO;
 
-    if (_videoWriter.status != AVAssetWriterStatusUnknown) {
```

Comment:
can you add a comment explaining that it's not necessary to check `AVAssetWriterStatusUnknown` anymore since we call startWriting immediately, even before video sample callback (please rephrase this).

Reply:
ok, but such a comment loses context here because in sources it is not visible that there was such a check, only in this diff